### DISCUSSION TITLE
群晖重启后数据和账户一同消失了？

### DISCUSSION THREAD

**Discussion by leoneoo** [usememos/memos#198] [Q&A]
群晖重启后数据和账户一同消失了？是不是我哪里没有配置好？

**Comment by boojack** [comment]
1. 看看启动后的模式是不是 prod
   <img width="1449" alt="image" src="https://user-images.githubusercontent.com/24653555/188320771-b375c02e-9172-456f-85e6-3ee3dc3b6b5c.png">

3. 检查是否将本地路径挂载进去了
   <img width="852" alt="image" src="https://user-images.githubusercontent.com/24653555/188320720-a30fe22e-5ed3-4aa5-a518-38503ba3d466.png">
